viewer/archivebotviewer/templates/item.html

```diff
@@ -23,6 +24,13 @@
                     {% end %}
                 </td>
                 <td class="numeric">{{ size }}</td>
+                <td>
+                    <a href="https://archive.org/download/{{ identifier }}/{{ filename }}">download</a>
+                    {% if job_ident %}
+                        &nbsp;
+                        <a href="{{ reverse_url('job', job_ident) }}">item</a>
```

Comment:
It might make more sense to make the filename anchor in `/item/ITEM` be the "download" link; however, I didn't want to mess with any habits.